lib/matplotlib/axes/_axes.py

```diff
@@ -2566,7 +2566,7 @@ def broken_barh(self, xranges, yrange, **kwargs):
             The x-positions and extends of the rectangles. For each tuple
             (*xmin*, *xwidth*) a rectangle is drawn from *xmin* to *xmin* +
             *xwidth*.
-        yranges : (*ymin*, *ymax*)
+        yranges : (*ymin*, *yheight*)
```

Comment:
The argument is named `yrange` (without `s`).

Reply:
Yes you are right! The original argument was wrong also. Let me do this again and also correct that. Do I need to open a new PR, or I can just use this if I push a new commit? @ImportanceOfBeingErnest

Reply:
Done. Pushed commit. I think it's automatically updated. Power of GitHub!